### DISCUSSION TITLE
[HELP] - Create a simple/basic "Task Template"

### DISCUSSION THREAD

**Discussion by Freebase394** [semaphoreui/semaphore#1409] [Q&A]
Hello everyone, 
I am new here! 
But I am not familiar with this softwer. 
It seems to have a nice interface and it is a realy simple web app, but I don't know how to run basic things like in Ubuntu 20.04 "apt-get update" or "apt-get upgrade -y"

Could someone create me some multiple examples please! And in a near feature it will be usefull to add to the docs app.

Best regards

**Comment by ansibleguy** [comment]
There is the need for documentation of such intro-examples.

Also: If you want to use Semaphore and want to start out right - make sure to read into Ansible (the program actually executed by Semaphore) first:
* [Getting started guide](https://docs.ansible.com/ansible/latest/getting_started/index.html)
* [Directory layout/structure](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#directory-layout)
* [Best practices](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html)

To your specific question: you could use the [ansible.builtin.apt](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html) module to interact with apt!

I might give writing Intro-Docs a shot this weekend.. if the time is in my favor (;
